From 3221c46961b4dcc88791f4a80dc8bd503c6cfe00 Mon Sep 17 00:00:00 2001
From: Matt Flax <flatmax@flatmax.org>
Date: Wed, 8 Mar 2017 21:13:24 +1100
Subject: [PATCH 032/454] ASoC: bcm2835_i2s.c: relax the ch2 register setting
 for 8 channels

This patch allows ch2 registers to be set for 8 channels of audio.
---
 sound/soc/bcm/bcm2835-i2s.c | 1 +
 1 file changed, 1 insertion(+)

--- a/sound/soc/bcm/bcm2835-i2s.c
+++ b/sound/soc/bcm/bcm2835-i2s.c
@@ -312,6 +312,7 @@ static int bcm2835_i2s_hw_params(struct
 
 	switch (params_channels(params)) {
 	case 2:
+	case 8:
 		format = BCM2835_I2S_CH1(format) | BCM2835_I2S_CH2(format);
 		format |= BCM2835_I2S_CH1(BCM2835_I2S_CHPOS(ch1pos));
 		format |= BCM2835_I2S_CH2(BCM2835_I2S_CHPOS(ch2pos));
